nntrainer/utils/util_func.cpp

```diff
@@ -197,4 +197,12 @@ bool istrequal(const std::string &a, const std::string &b) {
   });
 }
 
+char *get_realpath (const char *__restrict __name, char *__restrict __resolved) {
```

Comment:
This is trivial but stick to use camel case for function name instead of snake case?

Reply:
Thanks, I will reflect the modification.